app/frontend/src/components/Bar/BarWithHelp.tsx

```diff
@@ -26,7 +26,7 @@ export default function BarWithHelp({
   description,
 }: BarWithHelpProps) {
   const classes = useStyles();
-  return (
+  return (process.env.REACT_APP_API_BASE_URL ? <></> :
```

Comment:
```typescript
  return (process.env.REACT_APP_IS_BETA_ENABLED ? // scorebar stuff
    : null
```
Shouldn't it be referencing the feature flag instead? Also it's React convention to return `null` if we don't want to render anything at all.

Reply:
Oops